docs/using.md

```diff
@@ -29,20 +29,20 @@ This section contains information about basic virtual machine and OpenShift mana
 <a name="starting-openshift"></a>
 ### Starting OpenShift
 
-The [minishift start](./docs/minishift_start.md) command can be used to start your OpenShift instance.
+The [minishift start](minishift_start.md) command is used to start your OpenShift instance.
```

Comment:
In that case, I will make these corrections to the minishift .md too. I picked this style from that doc, so will change it there too to keep it consistent.